.github/workflows/bin-ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     outputs:
       go-version: ${{ steps.get-go-version.outputs.go-version }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```

Comment:
I see the versions updated to git commits... did GHA change from their versioning system? If not did you need some new feature or something.

A comment or mention in the commit message might help here.

Reply:
Using the commit hash for referencing actions is a new supply chain security requirement for our repositories. Tags are mutable, commits are not. There are automated compliance checks that do this for us. There is also an internal tool called `tsccr-helper` that we can use to pin the actions to commits. This is all internal machinery so I can't link to it here, but LMK if you want the references.